keyboards/atreus62/keymaps/d4mation/keymap.c

```diff
@@ -0,0 +1,194 @@
+#include QMK_KEYBOARD_H
+#include "d4mation.h"
+
+#define _DVR 0
+#define _QWR 1
+#define _LOWER 2
+#define _RAISE 3
+#define _NUM 4
+#define _ADJUST 16
```

Comment:
Non-consecutive layer numbers will generate extra empty layers in between, unnecessarily increasing flash usage. The better way to define layers is with an enum, eliminating the need to number them manually:

```c
enum layer_names {
    _DVR,
    _QWR,
    _LOWER,
    _RAISE,
    _NUM,
    _ADJUST
};
```

Reply:
Wow, yeah, that decreased my storage by 5%! That makes sense now that I think about it. I am fairly new to C and I had copied what I had there from somewhere else I think.

Just pushed that change to my branch.